### ISSUE TITLE
My progress got reseted :(

### ISSUE THREAD

**Issue by MrBunnyBunn** [Mezeman1/idle-ant-farm#50]
When the game updated it brought me back to the beginning, no achieves, no prestige, no equipment, etc

**Comment by Mezeman1** [comment]
I have not reset anything, you sure you logged in correctly?

**Comment by MrBunnyBunn** [comment]
Yeah I can't get it back but it's fine I wasn't really that far in the game anyways just thought I should let you know for bug purposes and stuff I probably didn't do something right

**Comment by MrBunnyBunn** [comment]
@Mezeman1  It happened again I'm at  the beginning again I got to I think fire ants, mining level 50ish and some higher attack levels n stuff? maybe further so I dunno I made sure I logged in correctly and everything :/

**Comment by Mezeman1** [comment]
Can you send me over, what e-mail you were using?

You're the only one having this issue, thusfar.

**Comment by Mezeman1** [comment]
Are you on discord? You might want to DM, so that other people don't see your email

**Comment by MrBunnyBunn** [comment]
You have your email account linked I emailed you so you can see what my email is idk if you'll get it because I hooked it up with my school account
I'll give you my discord this evening if it don't go through
